easypermissions/src/main/java/pub/devrel/easypermissions/helper/ActivityPermissionHelper.java

```diff
@@ -9,6 +9,7 @@
 /**
  * Permissions helper for {@link Activity}.
  */
+@Deprecated
 class ActivityPermissionHelper extends BaseFrameworkPermissionsHelper<Activity> {
```

Comment:
This class uses the framework fragments so I deprecated it... not sure if we actually want to do that though.